app/src/webview/java/org/mozilla/focus/webview/FocusWebViewClient.java

```diff
@@ -263,6 +264,27 @@ public void onReceivedSslError(WebView view, SslErrorHandler handler, SslError e
         }
     }
 
+    private void recordSSLErrors(SslError error) {
```

Comment:
What does the sslerror.toString return? I'd assume it gives a string representation of the error so we don't need to translate these to strings ourselves.